.gitignore

```diff
@@ -1,5 +1,7 @@
-KituraCache.xcodeproj
+*.xcodeproj/*
 Packages
-.build
+.build/*
 build
+.build-ubuntu/*
```

Comment:
This can be removed as well or we could add a [docker-compose.yml](https://github.com/IBM-Swift/Kitura/blob/master/docker-compose.yml) that uses it

Reply:
Sure I’ll remove it now, as I don’t think anyone is planning on adding a docker-compose.yml right now.